src/executables/sysrepoctl.c

```diff
@@ -184,6 +194,8 @@ new_iitem(const char *optarg, sr_install_mod_t **iitems, uint32_t *iitem_count)
     ++(*iitem_count);
 
     iitem->schema_path = optarg;
+    /*memcpy(iitem->module_ds.plugin_name, sr_module_ds_default.plugin_name, sizeof(sr_module_ds_t));*/
```

Comment:
> I wasn't able to compile when trying to access sr_module_ds_default from common.h directly.

I didn't clarify properly. Compiling is OK, but linking is the issue. I get the following error:

```
/usr/bin/ld: CMakeFiles/sysrepoctl.dir/src/executables/sysrepoctl.c.o: warning: relocation against `sr_module_ds_default' in read-only section `.text.startup'
/usr/bin/ld: CMakeFiles/sysrepoctl.dir/src/executables/sysrepoctl.c.o: in function `main':
sysrepoctl.c:(.text.startup+0x682): undefined reference to `sr_module_ds_default'
/usr/bin/ld: warning: creating DT_TEXTREL in a PIE
collect2: error: ld returned 1 exit status
```

I figured out that using the `API` macro will make `sr_module_ds_default` visible, but I'm not sure this is what you want. I don't know if `sr_module_ds_default` _should_ be visible to the external API or not.

Also, adding `common.c` to `SYSREPOCTL_SRC` causes a lot of other errors.

Reply:
> Also, adding common.c to SYSREPOCTL_SRC causes a lot of other errors.

You are right, not sure what I did before that it worked. Okay, so I guess an API function could be added that will return `const` pointer to the default datastore structure and it can then be used with ease in `sysrepoctl`.

Reply:
It looks better now with a proper API function.